scripts/dev

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eo pipefail
+
+KUBE_CONFIG=${KUBE_CONFIG:-$HOME/.kube}
+GCLOUD_CONFIG=${GCLOUD_CONFIG:-$HOME/.config/gcloud}
+EXTRA_DOCKER_PARAMS=${EXTRA_DOCKER_PARAMS:-}
+MARKETPLACE_TOOLS_TAG=${MARKETPLACE_TOOLS_TAG:-latest}
+MARKETPLACE_TOOLS_CONTAINER=${MARKETPLACE_TOOLS_CONTAINER:-gcr.io/cloud-marketplace-tools/k8s/dev}
+
```

Comment:
what do you think about the name `MARKETPLACE_TOOLS_IMAGE` instead?

Reply:
Opposed?

Reply:
Ah, missed this. Done. Thanks for the reminder!